src/keybindings.c

```diff
@@ -1827,11 +1827,11 @@ static void goto_matching_brace(GeanyDocument *doc)
 
 static gboolean cb_func_clipboard_action(guint key_id)
 {
-	GeanyDocument *doc = document_get_current();
+	GtkWidget *focusw = gtk_window_get_focus(GTK_WINDOW(main_widgets.window));
 
-	if (doc == NULL)
+	if (!IS_SCINTILLA(focusw)) {
```

Comment:
> @b4n I believe this is because the signal for the find or search and replace dialog is not captured there.

No the search *dialog* is no currently handled, only things in the main window are.  But I was talking about the *toolbar* search entry, which is in the main window ;)

And I'm talking about `cb_func_clipboard_action()`, which does work, not `cb_func_select_action()` which indeed didn't (but for `SELECT_ALL`).